js/src/ui/TxList/TxRow/txRow.js

```diff
@@ -165,6 +282,48 @@ class TxRow extends Component {
 
     return `/addresses/${address}`;
   }
+
+  getCondition = () => {
+    const { blockNumber, tx } = this.props;
+    let { time, block } = tx.condition;
+
+    if (time) {
+      if ((time.getTime() - Date.now()) >= 0) {
+        return `${dateDifference(new Date(), time, { compact: true })} left`;
+      } else {
+        return 'submitting';
+      }
+    } else if (blockNumber) {
+      block = blockNumber.minus(block);
+      return (block.toNumber() < 0)
+        ? block.abs().toFormat(0) + ' blocks left'
+        : 'submitting';
```

Comment:
Just make the strings here translatable. (Also line 247 is still, as part of the value for `which`, non-translatable)